src/redux/actions/profile/fetchProfile.js

```diff
@@ -0,0 +1,53 @@
+import { toast } from 'react-toastify';
+import * as profileActionCreators from './profileAction';
+import config from '../../../config.json';
+
+const { apiUrl } = config;
+const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NiwiZW1haWwiOiJ1c2VyZm91ckBnbWFpbC5jb20iLCJ1c2VybmFtZSI6InVzZXJmb3VyIiwicm9sZSI6Im5vcm1hbCIsInZlcmlmaWVkIjp0cnVlLCJpYXQiOjE1NzA2OTkyNjQsImV4cCI6MTU3MDc4NTY2NH0.2b1Qtkjd7PCaeV9AiydE5xP7YVAgP0xAAVSj14qS3nE';
+export const fetchProfile = () => (dispatch) => {
+  dispatch(profileActionCreators.fetchProfilePending());
+  fetch(`${apiUrl}/profile`, {
```

Comment:
I think this url should be in actionTypes as BASE_URL

Reply:
@habinezadalvan I have fixed these.